handbook/engineering/on_call/index.md

```diff
@@ -140,7 +140,13 @@ Open the alert UI to click on the check URL that was failing and verify it's now
 
 ### Useful dashboards
 
-[Lightstep dashboard for endpoint latency](https://app.lightstep.com/sourcegraph-prod/dashboard/gC24vW3g?range=3600)
+Check out the [kubectl cheatsheet](../deployments.md#kubectl-cheatsheet) for how to get access to Jaeger locally.
+
+### Access pod logs in GCP console
+
+1. Go to **Kubernetes Engine > Workloads**, then search and click on the pod you're interested, e.g. `sourcegraph-frontend`.
```

Comment:
Is it not possible to just document the direct link?

Reply:
The URL contains the `project` and `organizationId`, is that safe to be public?

Reply:
If we trust that Google implements authentication correctly, it should be fine. It might make us more vulnerable to some social engineering attack? I don't know, let's just merge as-is.